core/domain/state_domain.py

```diff
@@ -1711,23 +1711,6 @@ def delete_content_id_for_translation(self, content_id):
         else:
             self.translations_mapping.pop(content_id, None)
 
-    def get_translation_counts(self):
```

Comment:
Just to confirm, this wasn't used anywhere else, right?

Reply:
I confirm that this method is only used for populating translation counts in ExplorationOpportunitySummaryModel.